odoo/fields.py

```diff
@@ -812,6 +816,7 @@ def get_description(self, env):
     _description_groups = property(attrgetter('groups'))
     _description_change_default = property(attrgetter('change_default'))
     _description_group_operator = property(attrgetter('group_operator'))
+    _description_default_export_for_import = property(attrgetter('default_export_for_import'))
```

Comment:
Do we need the whole property boilerplate ? Can't we just add this parameter to ``Model._valid_field_parameter()`` method ? It allows to validate some custom parameter without actually doing anything with it if I am not wrong.

Reply:
I think that's not the case here because the export process uses `fields_get` (to benefit from the filtering and ACL-checking and stuff that that does), so the information it gets by default is only what's exported by `get_description`.

Though technically it could go and check the field data directly as well, if necessary. It really depends whether we want to send this information to the client, which here... is exactly what we're doing.